recipes/microed-tools/run_test.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/sh
+
+"${PREFIX}/bin/idoc2smv"                      \
```

Comment:
```suggestion
"idoc2smv"                      \
```
Your tests shouldn't require you to provide the absolute path of the binaries to work correctly. Also, this is the wrong prefix for testing. The test environment prefix is `CONDA_PREFIX`; the test environment is separate from the build/install environment so we can test that your package is relocatable.

Reply:
Fixed in f812a275efef3df391b7c2ccf74ba0f1067b7afe.

Reply:
Also added a separate `noarch` package for the test data in 372d3010b0c5519f53703441ce8554058bcd0385 to avoid duplicating 9 MiB of identical data across all architectures.